### PR TITLE
Fix skipping of Universe Tests in GitHub Actions

### PR DESCRIPTION
**Pull request by chetangudisagar**
## Overview

Description: The Universe Tests were getting skipped earlier unintentionally even after mentioning the maven build profile. Adding execution id and execution goals ensured that they are not skipped.

Why should this be merged: This resumes the universal tests in the GitHub Actions.

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -93,4 +93,4 @@ jobs:
         run: ./mvnw clean install -DskipTests -Pdocker -Dmaven.javadoc.skip=true -T 1C
 
       - name: Run universe tests
-        run: ./mvnw -pl :universe verify -Puniverse -DskipTests -Dmaven.javadoc.skip=true
+        run: ./mvnw -pl :universe verify -Puniverse -Dmaven.javadoc.skip=true
```
Comment (xnull) [inline .github/workflows/pull_request.yml:96]: If you enable unit tests, it will run unit tests along with integration tests, which is not desirable

**Review by xnull — CHANGES_REQUESTED** [review]

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (xnull) [reply .github/workflows/pull_request.yml:96]: sorry, looks like you are right

File: it/pom.xml [hunk 2]
```diff
@@ -100,6 +100,15 @@
...
                         <version>3.0.0-M5</version>
+                        <executions>
+                            <execution>
+                                <id>integration-test</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
```
Comment (xcchang) [inline it/pom.xml:111]: Could you explain why is this needed? for me it looks like the change in `pull_request.yml` will bring back the universe test?

**Review by xnull — APPROVED** [review]
-DskipTests should affect only the surefire plugin, for some reason it also offs failsafe plugin.
Surefire and failsafe plugins are quite messed in maven. 
It would be great to migrate to Gradle, that will improve everything in our build.

good catch, thank you

File: it/pom.xml (hunk 2 above)
Comment (xnull) [reply it/pom.xml:111]: This adds integration-test task into the verification task, it's a necessary change
P.S. as I remember universe tests worked even without this change though, but it's good to have it here

**Review by zhangn49 — APPROVED** [review]

File: it/pom.xml (hunk 2 above)
Comment (chetangudisagar) [reply it/pom.xml:111]: Without this "integration-test" goal, the integration-tests were getting skipped with the "universe" profile id. It used to work before with the "it" profile id. Hence I had to add this goal specifically.

**Review by PavelZaytsev — APPROVED** [review]
